src/autogluon/cloud/predictor/timeseries_cloud_predictor.py

```diff
@@ -198,6 +204,7 @@ def predict_real_time(
             target=self.target_column,
             static_features=static_features,
             accept=accept,
+            inference_kwargs=dict(known_covariates=known_covariates, **kwargs),
```

Comment:
I validated that this works locally but couldn't update the [unit tests](https://github.com/autogluon/autogluon-cloud/blob/master/tests/unittests/timeseries/test_timeseries.py#L47-L52) because currently `known_covariates` can only work with real-time prediction (and not batch prediction). One possible solution would be to train two predictors: one that expects `known_covariates` and one that doesn't, but I think that would be too much of an overhead.

Reply:
We might want to revisit this PR https://github.com/autogluon/autogluon-cloud/pull/113 for batch prediction support.